lib/pages/inventory.dart

```diff
@@ -1,8 +1,86 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:scibble/models/product.dart';
+import 'package:scibble/redux/inventory/actions.dart';
+import 'package:scibble/redux/store.dart';
+import 'package:scibble/theme/scibble_color.dart';
 
 class Inventory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Inventar');
+    return StoreConnector<AppState, _InventoryViewModel>(
+      converter: (store) => _InventoryViewModel(store.state.inventory,
+          getInventory: () => Future(store.dispatch(getInventory(store)))),
+      builder: (_, viewModel) => InventoryViewModel(
+        viewModel: viewModel,
+      ),
+      onInit: (store) => store.dispatch(getInventory(store)),
+    );
+  }
+}
+
+class InventoryViewModel extends StatelessWidget {
+  final _InventoryViewModel viewModel;
+
+  InventoryViewModel({Key key, this.viewModel}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return viewModel.inventory.length > 0
+        ? RefreshIndicator(
+            child: ListView.builder(
+              padding: EdgeInsets.all(5),
+              itemCount: viewModel.inventory.length,
+              itemBuilder: (BuildContext context, int index) {
+                return InventoryListTile(viewModel.inventory[index]);
+              },
+            ),
+            onRefresh: viewModel.getInventory,
+          )
+        : Center(
+            child: Column(children: <Widget>[
```

Comment:
If you add the key: value `mainAxisAlignment: MainAxisAlignment.center` to Column, it will center the column vertically.